lib/rubocop/cop/rspec/pending_block_inside_example.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for passing a block by `pending` or `skip` within examples.
+      #
+      # @example
+      #   # bad
+      #   it 'does something' do
+      #     pending 'not yet implemented' do
+      #       do_something
+      #     end
+      #   end
+      #
+      #   # good
+      #   it 'does something' do
+      #     pending 'not yet implemented'
+      #     do_something
+      #   end
+      #
+      #   # bad
+      #   it 'does something' do
+      #     skip 'not yet implemented' do
+      #       do_something
+      #     end
+      #   end
+      #
+      #   # good
+      #   it 'does something' do
+      #     skip 'not yet implemented'
+      #     do_something
+      #   end
+      #
+      #   # good - when outside example
+      #   pending 'not yet implemented' do
+      #   end
+      #
+      class PendingBlockInsideExample < Base
+        MSG = "Don't pass a block to `pending` or `skip` inside examples."
+
+        def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
+          return unless inside_example?(node)
+
+          add_offense(node) if %i[pending skip].include?(node.method_name)
```

Comment:
On an afterthought, this may be more performant:
```suggestion
          return unless %i[pending skip].include?(node.method_name)
          return unless inside_example?(node)

          add_offense(node)
```

Reply:
Thank you! I updated this PR.